### PR TITLE
Add logs to python getting started

### PR DESCRIPTION
**Pull request by svrnm**
following the instructions of [Logs Example](https://opentelemetry.io/docs/instrumentation/python/automatic/logs-example/), I was able to add logging capabilities to the python getting started 

---

**Preview**: https://deploy-preview-3360--opentelemetry.netlify.app/docs/instrumentation/python/getting-started/

### CODE REVIEW

**Review by cartermp — APPROVED** [review]
LGTM modulo conflicts

**Comment by cartermp** [comment]
@open-telemetry/python-approvers can we get a look?

File: content/en/docs/instrumentation/python/getting-started.md
Comment (chalin) [inline content/en/docs/instrumentation/python/getting-started.md:724]:
```suggestion
on.
```

**Review by chalin — APPROVED** [review]
LGTM aside from one copyedit, and the conflicts that will need to be resolved.

**Comment by svrnm** [comment]
@open-telemetry/python-approvers PTAL!

**Comment by cartermp** [comment]
Since this has several approvals and this is relatively straightforward and stable code, I'll merge.

**Pull request merged**
